sdk/python/kfp/compiler/compiler_test.py

```diff
@@ -553,7 +552,7 @@ def test_use_task_final_status_in_non_exit_op_yaml(self):
 - {name: message, type: PipelineTaskFinalStatus}
 implementation:
   container:
-    image: python:3.7
+    image: python:3.9
```

Comment:
I don't think we need to change this. Our default image for lightweight component is still Python:3.7

Reply:
Good call. I misunderstood this template definition as having a functional effect for our tests.